src/popup/pages/Send/index.js

```diff
@@ -541,9 +546,9 @@ class SendPage extends React.Component {
       return
     }
 
-    feeAmount = feeAmount || 0
+    feeAmount = feeAmount || this.pageConfig.defaultFeeAmount
```

Comment:
Is the `||` everywhere intentional, or is it usefully replaced by `??`? Also, if there were a linter, it'd probably catch this.

Reply:
it's from an input, so I think it's `""` in the default state